### PR TITLE
Windows Logstash connector for Palo Alto Firewalls Custom Tables

### PR DESCRIPTION
**Pull request by mzubeir**
Required items, please complete
   
   Change(s):
   - New JSON for Windows Logstash ingestion to Azure Sentinel allowing Mod Palo Alto Data connector to read from Custom Table.

   Testing Completed:
   - This is the existing Palo Alto Data connector which has been modified to point to the custom tables in LogAnalytics.  This has been tested an working.

LogStashPANCL
LogStash PAN OS logs to Azure Sentinel for Modified Custom Tables For Windows only Platforms/Infrastructure where no Linux syslog servers exist or can be used.
This Data connector has been modified to point to the custom tables in the Azure Log Analytics table. This addresses issues when using Windows with Logstash to feed logs from PAN's. These go into a custom Table which the existing PAN Connector points to the CommonSecurityLog Tables so does not go green.

Refer to the orignal PaloAlto Connector: https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors/Palo%20Alto%20Networks

Compare the Table Headers etc. and amend according to your table name etc. All references to the Tables etc. have an *_CL and columns have _s.

### CODE REVIEW

File: DataConnectors/SsPaloAlto.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "\nPaloAltoNetworks_CL\n| where DeviceVendor_s == \"Palo Alto Networks\"\n| where DeviceProduct_s has \"PAN-OS\"\n\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "PaloAltoNetworks (PaloAlto)",
```
Comment (v-webr) [inline DataConnectors/SsPaloAlto.json:33]: Name must be PaloAltoNetworks_CL

**Comment by v-sabiraj** [comment]
Hey @mzubeir, please update the requested changes, thanks.

**Comment by v-sabiraj** [comment]
Hey @mzubeir, please update the requested changes, thanks.

**Comment by v-sabiraj** [comment]
Hey @mzubeir, please update the requested changes, thank you.

**Comment by v-sabiraj** [comment]
Hey @mzubeir, please update the requested changes, thank you.

**Comment by v-sabiraj** [comment]
Hey @mzubeir, will you please update the requested changes, thank you.

**Comment by v-sabiraj** [comment]
Hey @mzubeir, will you please update the requested changes, thank you.

**Comment by v-sabiraj** [comment]
Hey @mzubeir, please add sample data table "PaloAltoNetworks_CL", thanks.

**Comment by v-sabiraj** [comment]
Hey @mzubeir, please add sample data table "PaloAltoNetworks_CL", and resolve other comments, thanks.

**Comment by vmanojreddy** [comment]
Hey @lstajcic, we wanted to check on the status of this PR [Windows Logstash connector for Palo Alto Firewalls Custom Tables #4302](https://github.com/Azure/Azure-Sentinel/pull/4302). It is pending from more than 60 days. Let us know if any assistance is required for this PR. Per our standard operating procedures if no response is received in the next 7 business days we will close this PR. Thank you for your cooperation.

**Comment by v-sabiraj** [comment]
Hey @mzubeir, please check this comment, thanks.

> Hey @[mzubeir](https://github.com/mzubeir) ,we wanted to check on the status of this PR [Windows Logstash connector for Palo Alto Firewalls Custom Tables #4302](https://github.com/Azure/Azure-Sentinel/pull/4302). It is pending from more than 60 days. Let us know if any assistance is required for this PR. Per our standard operating procedures if no response is received in the next 7 business days we will close this PR. Thank you for your cooperation.

**Comment by vmanojreddy** [comment]
Hi @mzubeir  Since we have not received response from last 7 days, we are closing your PR [Windows Logstash connector for Palo Alto Firewalls Custom Tables #4302](https://github.com/Azure/Azure-Sentinel/pull/4302) per our standard operating procedures. If you still need support for this issue you can re-open the PR at any time.
If you do re-open, we simply request that you ensure the PR has response to the last request. Thank you for your cooperation.

**Pull request closed without merging**